.github/workflows/reusable_functional_tests.yml

```diff
@@ -0,0 +1,209 @@
+name: Compilation & tests
+
+on:
+  workflow_call:
+    inputs:
+      branch_for_exchange:
+        required: false
```

Comment:
I think all these `required` should be `true` (they're absolutely needed for the workflow to run), however they are not "required to be set by the user", as there is a default value.
The doc is unclear on that, but that's how I understand and implemented it on the [ETH composite action.](https://github.com/LedgerHQ/app-ethereum/blob/develop/.github/actions/commit-changes/action.yml)

Reply:
Done

Reply:
Reverted :(